src/test/java/ay2021s1_cs2103_w16_3/finesse/logic/commands/bookmarkcommands/ConvertBookmarkIncomeCommandTest.java

```diff
@@ -14,97 +14,97 @@
     public void execute_invalidIndexUnfilteredList_throwsCommandException() {
-        Index outOfBoundIndex = Index.fromOneBased(model.getFilteredFrequentIncomeList().size() + 1);
+        Index outOfBoundIndex = Index.fromOneBased(model.getFilteredBookmarkIncomeList().size() + 1);
         Date dateOfConvertedIncome = new Date(VALID_DATE_SPOTIFY_SUBSCRIPTION);
 
-        ConvertFrequentIncomeCommand convertFrequentIncomeCommand =
-                new ConvertFrequentIncomeCommand(outOfBoundIndex, dateOfConvertedIncome);
+        ConvertBookmarkIncomeCommand convertBookmarkIncomeCommand =
+                new ConvertBookmarkIncomeCommand(outOfBoundIndex, dateOfConvertedIncome);
 
-        assertCommandFailure(convertFrequentIncomeCommand, model, MESSAGE_INVALID_FREQUENT_INCOME_DISPLAYED_INDEX);
+        assertCommandFailure(convertBookmarkIncomeCommand, model, MESSAGE_INVALID_BOOKMARK_INCOME_DISPLAYED_INDEX);
     }
 
     @Test
     public void execute_validIndexFilteredList_success() {
-        showFrequentIncomeAtIndex(model, INDEX_FIRST);
+        showBookmarkIncomeAtIndex(model, INDEX_FIRST);
 
         Date dateOfConvertedIncome = new Date(VALID_DATE_SPOTIFY_SUBSCRIPTION);
 
-        FrequentIncome frequentIncomeToBeConverted = model.getFilteredFrequentIncomeList()
+        BookmarkIncome bookmarkIncomeToBeConverted = model.getFilteredBookmarkIncomeList()
                 .get(INDEX_FIRST.getZeroBased());
-        ConvertFrequentIncomeCommand convertFrequentIncomeCommand =
-                new ConvertFrequentIncomeCommand(INDEX_FIRST, dateOfConvertedIncome);
+        ConvertBookmarkIncomeCommand convertBookmarkIncomeCommand =
+                new ConvertBookmarkIncomeCommand(INDEX_FIRST, dateOfConvertedIncome);
 
         Model expectedModel = new ModelManager(model.getFinanceTracker(), new UserPrefs());
-        Income convertedIncome = frequentIncomeToBeConverted.convert(dateOfConvertedIncome);
+        Income convertedIncome = bookmarkIncomeToBeConverted.convert(dateOfConvertedIncome);
         expectedModel.addIncome(convertedIncome);
 
-        String expectedMessage = String.format(ConvertFrequentIncomeCommand.MESSAGE_CONVERT_FREQUENT_INCOME_SUCCESS,
+        String expectedMessage = String.format(ConvertBookmarkIncomeCommand.MESSAGE_CONVERT_BOOKMARK_INCOME_SUCCESS,
                 convertedIncome);
 
-        assertCommandSuccess(convertFrequentIncomeCommand, model, expectedMessage, expectedModel, true);
+        assertCommandSuccess(convertBookmarkIncomeCommand, model, expectedMessage, expectedModel, true);
     }
 
     @Test
     public void execute_invalidIndexFilteredList_throwsCommandException() {
-        showFrequentIncomeAtIndex(model, INDEX_FIRST);
+        showBookmarkIncomeAtIndex(model, INDEX_FIRST);
 
         Index outOfBoundIndex = INDEX_SECOND;
         Date dateOfConvertedIncome = new Date(VALID_DATE_SPOTIFY_SUBSCRIPTION);
         // ensures that outOfBoundIndex is still in bounds of frequent income list
```

Comment:
The edits for this file mirror that of `ConvertBookmarkExpenseCommandTest.java`.